### PR TITLE
Nav updates

### PR DESCRIPTION
**Pull request by robbiecarlton**

### CODE REVIEW

**Comment by robbiecarlton** [comment]
@levity - good to merge?

**Comment by robbiecarlton** [comment]
@levity - btw, couldn't test the full width version because I don't have a wide enough screen. Don't anticipate any problems, but could you double check?

File: src/containers/App.js [hunk 1]
```diff
@@ -35,7 +36,7 @@ const makeNavLinks = (currentUser, community) => {
     {url: network && `/n/${network.slug}`, icon: 'merkaba', label: 'Network'},
     {url: slug && url('about'), icon: 'Help', label: 'About'},
     {url: canInvite(currentUser, community) && url('invite'), icon: 'Mail', label: 'Invite'},
-    {url: canModerate(currentUser, community) && url('settings'), icon: 'Settings', label: 'Settings'}
+    {url: canModerate(currentUser, community) && url('settings'), icon: 'Settings', label: 'Admin Settings'}
```
Comment (levity) [inline src/containers/App.js]: we settled on "Community Settings"

File: src/containers/App.js [hunk 2]
```diff
@@ -158,12 +158,15 @@ export default class App extends React.Component {
         path={path}
         search={doSearch}
         opened={leftNavIsOpen}
-        networkNavAnimation={moveWithMenu}
-        networkNavEasing={leftNavEasing}
         />}
 
       <VelocityComponent animation={moveWithMenu} easing={leftNavEasing}>
         <div id='main'>
+          {currentUser && !isMobile && networkCommunities &&
```
Comment (levity) [inline src/containers/App.js]: this is a lot of clauses; can we just wrap them up into a `showNetworkNav` const?

**Comment by levity** [comment]
I'm noticing that the extra menu item fits better when the menu is closed than when it's open. We should be factoring that into deciding how many links to show up there (which we can do with CSS since the main div gets the `leftNavIsOpen` class).

I would reduce the max-width of the community name from 400px to 240px also

**Comment by robbiecarlton** [comment]
@levity - I have to stop for today, didn't get a chance to finish the variable width depending on leftNavIsOpen. 
If this looks good otherwise, can you merge it, to fix the profile bug ed was talking about.
I'll finish the variable width stuff tomorrow

**Comment by levity** [comment]
sounds good

**Pull request merged**